inc/CLI/ProjectCommand.php

```diff
@@ -0,0 +1,224 @@
+				'loader_instance'       => $loader_instance,
+			);
+
+			WP_CLI::line( json_encode( $info ) );
+		} else {
+			WP_CLI::line( "Project ID:\t\t" . $project_id );
+			WP_CLI::line( "Project name:\t\t" . $project_name );
+			WP_CLI::line( "Project slug:\t\t" . $project_slug );
+			WP_CLI::line( "Repository Cache:\t" . $local_path );
+			WP_CLI::line( "Repository URL:\t\t" . $repository_url );
+			WP_CLI::line( "Repository Type:\t" . $repository_type );
+			WP_CLI::line( "Repository VCS Type:\t" . $repository_vcs_type );
+			WP_CLI::line( "Repository Visibility:\t" . $repository_visibility );
+			WP_CLI::line( "Repository SSH URL:\t" . $repository_ssh_url );
+			WP_CLI::line( "Repository HTTPS URL:\t" . $repository_https_url );
+			WP_CLI::line( "Repository Instance:\t" . $repository_instance );
+			WP_CLI::line( "Loader Instance:\t" . $loader_instance );
+		}
+	}
+
+	/**
+	 * List language packs for the given project.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <project>
+	 * : Path or ID of the project.
+	 *
+	 * [--field=<field>]
+	 * : Prints the value of a single field for each language pack.
+	 *
+	 * [--fields=<fields>]
+	 * : Limit the output to specific language pack fields.
+	 *
+	 * [--format=<format>]
+	 * : Render output in a particular format.
+	 * ---
+	 * default: table
+	 * options:
+	 *   - table
+	 *   - csv
+	 *   - json
+	 *   - count
+	 *   - yaml
+	 * ---
+	 *
+	 * ## AVAILABLE FIELDS
+	 *
+	 * These fields will be displayed by default for each translation:
+	 *
+	 * * Language
+	 * * Updated
+	 * * English Name
+	 * * Native Name
+	 * * Package
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Display available language packs for the given project
+	 *     $ wp traduttore project list-language-packs 1 --fields=Language,Package
+	 *     +----------+----------------------------------------------------------------+
+	 *     | Language | Package                                                        |
+	 *     +----------+----------------------------------------------------------------+
+	 *     | fr_FR    | https://translate.example.com/content/traduttore/foo-fr_FR.zip |
+	 *     | de_DE    | https://translate.example.com/content/traduttore/foo-de_DE.zip |
+	 *     +----------+----------------------------------------------------------------+
+	 *
+	 * @subcommand list-language-packs
+	 *
+	 * @since 3.0.0
+	 *
+	 * @param array $args Command args.
+	 * @param array $assoc_args Associative args.
+	 */
+	public function list_language_packs( $args, $assoc_args ): void {
+		$locator = new ProjectLocator( $args[0] );
+		$project = $locator->get_project();
+
+		if ( ! $project ) {
+			WP_CLI::error( 'Project not found' );
+		}
+
+		$translation_sets = (array) GP::$translation_set->by_project_id( $project->get_id() );
+
+		$language_packs = [];
+
+		/* @var GP_Translation_Set $set */
+		foreach ( $translation_sets as $set ) {
+			/* @var GP_Locale $locale */
+			$locale = GP_Locales::by_slug( $set->locale );
```

Comment:
Because `$locale` is only being used in https://github.com/wearerequired/traduttore/pull/110/files#diff-04f5d18d9094fe2c3455feb2a6dc6fd9R203

But with the changes to this command it might change. The reasoning is to only define the variable short before it is being used.